flag.go

```diff
@@ -243,6 +244,16 @@ func (f *FlagSet) lookup(name normalizedName) *Flag {
 	return f.formal[name]
 }
 
+// Mark a flag deprecated in your program
+func (f *FlagSet) MarkDeprecated(name string, usageMessage string) error {
+	flag := f.Lookup(name)
```

Comment:
I can't even count of the person writing code to know the names of their flags?   ok, i'll look at it.